tools/ci/check-affected.py

```diff
@@ -44,70 +45,71 @@
 print(">>>> eof debug")
 
 # define affection actions -> add to projects arrays and generate preview link
+
+
 def affect_app(app, preview_name=None):
-  print(f"{app} is affected")
-  projects.append(app)
-  if not preview_name:
-    preview_name=app
-  previews[app] = f'https://{preview_name}.{args.branch_slug}.vega.rocks'
+    print(f"{app} is affected")
+    projects.append(app)
+    if not preview_name:
+        preview_name = app
+    previews[app] = f'https://{preview_name}.{args.branch_slug}.vega.rocks'
+
 
 # check appearance in the affected string for main apps
 for app in main_apps:
-  if app in affected:
-    affect_app(app)
+    if app in affected:
+        affect_app(app)
 
 # if non of main apps is affected - test all of them
 if not projects:
-  for app in main_apps:
-    affect_app(app)
+    for app in main_apps:
+        affect_app(app)
 
 # generate e2e targets
 projects_e2e = [f'{app}-e2e' for app in projects]
 
 # check affection for multisig-signer which is deployed only from develop and pull requests
 if args.event_name == 'pull_request' or 'develop' in args.github_ref:
-  if 'multisig-signer' in affected:
-    affect_app('multisig-signer', 'tools')
+    if 'multisig-signer' in affected:
+        affect_app('multisig-signer', 'tools')
 
 # now parse apps that are deployed from develop but don't have previews
 if 'develop' in args.github_ref:
-  for app in ['static', 'ui-toolkit']:
-    if app in affected:
-      projects.append(app)
+    for app in ['static', 'ui-toolkit']:
+        if app in affected:
+            projects.append(app)
 
 # if ref is in format release/{env}-{app} then only {app} is deployed
 if 'release' in args.github_ref:
-  for app in main_apps:
-    if f'{args.github_ref}'.endswith(app):
-      projects = [app]
-      projects_e2e = [f'{app}-e2e']
+    for app in main_apps:
+        if f'{args.github_ref}'.endswith(app):
+            projects = [app]
+            projects_e2e = [f'{app}-e2e']
 
 
 projects = json.dumps(projects)
 
-# The trading project does not use the deafult NX e2e setup (cypress)
-projects_e2e.remove('trading-e2e')
```

Comment:
This looks like mostly formatting changes, and you might want to rebase, but I'd like to make sure this line stays so that trading-e2e doesnt attempte to run